### DISCUSSION TITLE
Notification of end of untar processing

### DISCUSSION THREAD

**Discussion by comassky** [minio/minio#18445] [Q&A] [answered]
Hello everyone,

In my project we're using Minio's untar (snowball) feature.

From what we understand, we don't have any information about the end of the untar processing (we use the Change Data Capture pattern when depositing).

Does anything exist or is planned in Minio to meet this need?

**Comment by klauspost** [comment]
When the call returns the objects have been written. They are being written as the file is streamed in.

**Comment by comassky** [comment]
So when a file upload with the UNTAR option returns, it means that the untar went well and is finished ?

**Reply by klauspost** [reply, marked as answer]
Yes.
